### ISSUE TITLE
bug of loss.backward()

### ISSUE THREAD

**Issue by liujiahao11** [mperezcarrasco/PyTorch-EGBAD#1]
In your code, you train D first and then train G, but you didn't call optimizerG.zero_grad() before loss_g.backward().
This means the generator is optimized based on loss_g and loss_d.
